redash/query_runner/rockset.py

```diff
@@ -76,12 +87,23 @@ def __init__(self, configuration):
             'api_server', "https://api.rs2.usw2.rockset.com"))
 
     def _get_tables(self, schema):
-        for col in self.api.list():
-            table_name = col['name']
-            describe = self.api.query('DESCRIBE "{}"'.format(table_name))
-            columns = list(set(map(lambda x: x['field'][0], describe['results'])))
-            schema[table_name] = {'name': table_name, 'columns': columns}
-        return schema.values()
+        pool = ThreadPool(processes=10)
```

Comment:
In some configurations (and this is going to become the default) we run Redash's API server with gevent's monkeypatching. I'm not sure how "healthy" it is to mix threads with gevent. 

How about we land this PR without the threads optimization and then revisit this in a follow up?

Reply:
Hey @arikfr, thanks for the feedback. Given that future direction, would it make sense to replace this multithreaded approach with [grequests](https://github.com/spyoungtech/grequests)? I was originally leaning in that direction actually but figured this would be a less invasive change.

Reply:
At the moment the default deployment doesn't use gevent and we don't have a timeline for the switch. So for now I suggest we don't use anything that depends on it.